src/App.js

```diff
@@ -9,37 +9,41 @@ class App extends React.Component {
     super(props);
     this.state = {
       userSearch: '',
-      locationData: '',
+      locationData: {},
       renderMap: false,
     };
   };
 
-  getCityInfo = async (e) => {
+  handleSubmit = (e) => {
     e.preventDefault();
-    let userSearch = e;
-    userSearch.toLowerCase();
+    let city = e.target.city.value;
```

Comment:
Is the `.city` in this chain on the assignor side of this line necessary/ functional vs. just `e.target.value`?